Masquerade4997Library/src/main/java/Library4997/MasqControlSystems/MasqPurePursuit/MasqPositionTracker.java

```diff
@@ -19,21 +15,31 @@
 
 public class MasqPositionTracker implements MasqHardware {
     private MasqMotor xSystem, ySystem;
-    public BNO055IMU imu;
+    public MasqAdafruitIMU imu;
     private double globalX = 0, globalY = 0, prevX = 0, prevY = 0;
     private Orientation angles;
     private double zeroPos = 0;
+    private String imuName;
 
-    public MasqPositionTracker(MasqMotor xSystem, MasqMotor ySystem, BNO055IMU imu) {
-        this.imu = imu;
+    public MasqPositionTracker(MasqMotor xSystem, MasqMotor ySystem) {
+        this.xSystem = xSystem;
+        this.ySystem = ySystem;
+        imuName = "imu";
```

Comment:
BRUH. make initalizeIMU private. Then at the end of this constructor call intializeIMU(hardwareMap, imuName)